packages/discovery-react-components/package.json

```diff
@@ -43,6 +43,7 @@
     "react-virtualized": "9.21.1"
   },
   "devDependencies": {
+    "@types/pdfjs-dist": "^2.10.378",
```

Comment:
It seems that this [package](https://www.npmjs.com/package/@types/pdfjs-dist) is now deprecated, showing:
> This is a stub types definition. pdfjs-dist provides its own type definitions, so you do not need this installed.